### DISCUSSION TITLE
Blue contributions

### DISCUSSION THREAD

**Discussion by Krumuvecis** [community/community#42244] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

A few hours ago I noticed that contributions on profile pages are showing in blue instead of the usual green. Is it somehow due to Christmas? I searched the web and discussions, but couldn't find anything. What's up with the new color?

**Comment by fakhrads** [comment]
Me too. I think it's a new thing about 2023 from github

**Reply by nkayilovette03** [reply]
Or maybe a newbies got in and changed the color code to his preferred Blue 🤣

**Reply by DSDmark** [reply]
In that case, we need to report that newbie to GitHub support. 😂

**Comment by heissanjay** [comment]
better if its due to Christmas

**Reply by nkayilovette03** [reply]
https://github.com/orgs/community/discussions/42244#discussioncomment-4469268

**Comment by radar-blip** [comment, marked as answer]
I'm sure it's due to Christmas and/or New years. I remember they were orange on Halloween. 🎃

**Comment by FBH464** [comment]
looks good this color more striking but well if it's temporary it doesn't matter

**Comment by DSDmark** [comment]
Hi @Krumuvecis,

> GitHub often updates the design of its website and adds new features, including changes to the colors used to display various elements. It is possible that the change in the color of contributions on profile pages is related to a recent design update or a special event, such as Christmas.

You can check the GitHub blog or social media accounts to see if there are any announcements about recent updates or changes

Here is [**GitHub blog**](https://github.blog/).

Btw, the color will disappear when event end. So, if you like in that way. Then, you can use GitHub action for that, or you can use this, instead. :- [**pumpkin**](https://github.com/afonsopacifer/pumpkin).

**Comment by Tylerasa** [comment]
I believe it has to do with Christmas. When you inspect the graph, you'll notice that the style is set to winter, so I'm guessing it's something Christmas/winter-related, similar to what they did for Halloween.

![Screenshot (404)](https://user-images.githubusercontent.com/20054166/208870986-b4081d76-2199-4bd8-8a77-85d83d104d5d.png)

**Comment by MihirRajeshPanchal** [comment]
On GitHub, the blue contributions on a user's profile page represent the number of commits (i.e., saved changes) made to repositories (i.e., collections of code and other files) on GitHub by that user during the past year. The blue squares are used to visualize the contributions, with each square representing one commit on a particular day. The color of the square reflects the frequency of commits on that day, with shades of blue ranging from light to dark depending on the number of commits.

For example, if a user made three commits on a particular day, the square for that day would be a darker shade of blue than if the user had made only one commit on that day. The contributions chart is a way to visualize a user's activity on GitHub over time, and it can be a useful tool for identifying patterns in a user's work and for comparing the activity of different users.

GitHub also uses a similar system to visualize the contributions of organizations (i.e., groups of users working together on projects). In this case, the contributions chart shows the combined activity of all the members of the organization.

**Comment by satrio-pamungkas** [comment]
Actually, I don't remember we had similar color during winter last year. Perhaps this is a new thing.

**Comment by MakePixelsWork** [comment]
It was also orange when it was Halloween... so its probably a seasonal thing. That and the vars naming.

**Comment by kyrios-jawad** [comment]
maybe a new update? or seasonal thing

**Comment by hhsde** [comment]
With the potential snow bomb cyclone coming in over next few days in the US, I'd say it's a reference to that. But, it does coincide with the holidays. *shrug*

"... Jack Frost nipping at your nose ..."

**Comment by perisicnikola37** [comment]
Seasonal thing🎅

**Reply by DSDmark** [reply]
Like a `PUBG`. Ooh i get, it's a new update😅

**Comment by hernanmarino** [comment]
Github is obviously celebrating the Argentina World Cup : )   Why would they pick blue for Christmas / New Year otherwise ?

**Comment by SethMoffat** [comment]
I don't want to be a downer but but it looks worse visually and I think green was already a better holiday color.
I guess that's just GitHub being GitHub

**Comment by waheb-benzaid** [comment]
it's 12 AM my time, and it's green again !

**Comment by james-valente-simplisafe** [comment]
Maybe just for the day ...
December 21 = Winter Solstice

**Comment by sadafnajam** [comment]
I believe it was due to winter official start date ❄️

**Reply by waheb-benzaid** [reply]
yes I think so !

**Comment by jfullstackdev** [comment]
and I'm not seeing it on my profile or other profiles, maybe because there is no winter here, localization I think so

**Reply by kyrios-jawad** [reply]
its gone already

**Comment by Krumuvecis** [comment]
Thanks guys! I just returned from holidays, and am seeing green again. I guess it was for Christmas indeed!

**Comment by JayWelsh** [comment]
Mine is blue today, why?

**Reply by MihirRajeshPanchal** [reply]
It's summer solstice 21st June 2023

**Reply by JayWelsh** [reply]
Oh I see, probably blue then for winter solstice since I'm in the Southern Hemisphere. Thanks!

**Comment by JayWelsh** [comment]
It would be nice if there was some little badge or something which indicated what the event was (most are obvious, but things like summer/winter solstice are a little less obvious).
